index.js

```diff
@@ -5,9 +5,10 @@ var randomBytes = require('randombytes')
 var unorm = require('unorm')
 
 var DEFAULT_WORDLIST = require('./wordlists/en.json')
+var JAPANESE_WORDLIST = require('./wordlists/ja.json')
 
 function mnemonicToSeed(mnemonic, password) {
-  var mnemonicBuffer = new Buffer(mnemonic, 'utf8')
+  var mnemonicBuffer = new Buffer(unorm.nfkd(mnemonic), 'utf8')
   var saltBuffer = new Buffer(salt(password), 'utf8')
```

Comment:
Moved them together for consistency in https://github.com/bitcoinjs/bip39/commit/b7d3901adc7d9ff2303fd22be032fba7f3f26856